servers/rendering/renderer_rd/forward_clustered/render_forward_clustered.cpp

```diff
@@ -1142,7 +1142,7 @@ void RenderForwardClustered::_update_sdfgi(RenderDataRD *p_render_data) {
 		for (int i = 0; i < p_render_data->render_sdfgi_region_count; i++) {
 			sdfgi->render_region(rb, p_render_data->render_sdfgi_regions[i].region, p_render_data->render_sdfgi_regions[i].instances, exposure_normalization);
 		}
-		if (p_render_data->sdfgi_update_data->update_static) {
+		if (p_render_data->sdfgi_update_data != nullptr && p_render_data->sdfgi_update_data->update_static) {
```

Comment:
I would move the condition `p_render_data->sdfgi_update_data != nullptr` to the top of this function and early out if `p_render_data->sdfgi_update_data == nullptr` that way we can avoid the cost of the entire function.

Reply:
Done and I agree! I chose to add it there for safety reasons to not introduce unwanted side-effects since I'm not too familiar with the codebase yet. 😃

Reply:
I figured! I appreciate the caution :)